pkg/stub/stub.go

```diff
@@ -69,6 +69,14 @@ type RunPodInterface interface {
 	RunPodSandbox(context.Context, *api.PodSandbox) error
 }
 
+// UpdatePodInterface handles UpdatePodSandbox API requests.
+type UpdatePodInterface interface {
+	// UpdatePodSandbox relays an UpdatePodSandbox request to the plugin.
+	// TODO: Add comment on anything the plugin can do in response (if
+	// applicable).
+	UpdatePodSandbox(context.Context, *api.PodSandbox, *api.LinuxResources, *api.LinuxResources) (*api.PodSandboxUpdate, error)
```

Comment:
Considering the previous notes about pod-level cgroup management being done in the kubelet and not the runtime, I think the return part signature should be just an error here: `UpdatePodSandbox(context.Context, *api.PodSandbox, *api.LinuxResources, *api.LinuxResources) error`.

Reply:
Fixed this, thanks